yarn.lock

```diff
@@ -0,0 +1,3323 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
How and when does this file get updated / how do we keep it in sync?

Reply:
I can’t speak to `rules_nodejs`, but how it works in the outside world
is that `yarn.lock` changes only when you add, remove, or upgrade
dependencies. You do so by running `yarn add` (to add or upgrade) or
`yarn remove`, which updates `package.json` and `yarn.lock` as well as
actually downloading the package to `node_modules` if required.

Reply:
Yup, one is expected to call `yarn add` and `yarn remove` to add and remove a dependency, respectively.